layouts/index.html

```diff
@@ -33,7 +33,7 @@ <h2 class="title--answer">Mostly...</h2>
       <li class="list__item--ok">
         Desktop environment:
         <a href="https://www.gnome.org/">GNOME</a>,
-        <a href="https://kde.org/">KDE</a>
+        <a href="https://kde.org/plasma-desktop">Plasma</a>
```

Comment:
Use `KDE Plasma` otherwise people aren't going to know what you're talking about.

Reply:
+1 "KDE Plasma" seems to be the official name and it is a bit more explicit